### PR TITLE
Solving conversion of mails with Swiftmailer - The Second

### PR DESCRIPTION
**Pull request by adrichel**
We had issues with exceptions preventing parsing of faulty msg files - and we are in need of an option to have a lossy conversion instead on no conversion in such cases. So here is a new PR as the initial one has been stale for a while.

I took the changes @risototh kindly provided in their PR (https://github.com/hfig/MAPI/pull/24) and refactored it with changes @hfig suggested in their comment (https://github.com/hfig/MAPI/pull/24#issuecomment-860141464).

TLDR:  you can now set a property to the `Factory` class to define if the exceptions in `$message->toMime()` method are being muted like this: 
```php
$messageFactory = new MAPI\MapiMessageFactory(new Swiftmailer\Factory(muteConversionExceptions: true));
```

### CODE REVIEW

**Comment by adrichel** [comment]
Closing to reopen from a feature branch (was we need to commit more changes that would be out of scope of this pr)
